README.md

```diff
@@ -1,6 +1,8 @@
-# zenoh_autoware_v2x
-
+# Zenoh Autoware V2X
+## Architecture
```

Comment:
BTW, it will be better to have space around the title. Markdown lint won't complain.